tests/test_integration.py

```diff
@@ -0,0 +1,55 @@
+from datetime import datetime
+
+import seb
+from seb.cli import run_benchmark_cli
+from seb.registries import tasks
+
+
+@tasks.register("test-encode-task")
+def create_test_encode_task() -> seb.Task:
+    class DummyTask(seb.Task):
+        name = "test-encode-task"
+        main_score = "a_metric"
+        description = "NA"
+        reference = "NA"
+        version = "NA"
+        languages = []  # noqa: RUF012
+        domain = []  # noqa: RUF012
+        task_type = "Classification"
+
+        def evaluate(self, model: seb.Encoder) -> seb.TaskResult:
+            model.encode(["a test sentence"], task=self)
+
+            return seb.TaskResult(
+                task_name="test-encode-task",
+                task_description="NA",
+                task_version="NA",
+                time_of_run=datetime.now(),
+                scores={"en": {"a_metric": 1.0}},
+                main_score="a_metric",
+            )
+
+        def get_descriptive_stats(self) -> dict:
+            return {}
+
+    return DummyTask()
+
+
+def test_integration_dummy():
+    """Runs all sorts of models on a dummy task to see if they can run without breaking.
+    Cache is ignored so that the models are actually run.
+    """
+    models = [
+        "fasttext-cc-da-300",
+        "intfloat/e5-small",
+        "translate-e5-small",
+    ]
+    tasks = ["test-encode-task"]
+    run_benchmark_cli(models=models, tasks=tasks, ignore_cache=True)
+
+
+def test_integration_lcc():
+    """Runs model(s) on LCC to see if everything works in order with tasks included."""
+    models = ["sentence-transformers/all-MiniLM-L6-v2"]
```

Comment:
move up as parametrize

Reply:
Can do, but can you tell me why it's better?

Reply:
it removed the test case from how it is tested. It allows you to see which model failed instead of one of the three models failed.

Reply:
The only problem is that in that case we're not testing what happens when we run two models at the same time. But I guess that isn't a huge issue.

Reply:
testing multiple models should be a part of the cli test I believe - while it should be clear here which integrations fails.